eng/helix/helix.proj

```diff
@@ -24,6 +24,7 @@
     <IsExternal>true</IsExternal>
     <MaxRetryCount Condition="'$(MaxRetryCount)' == ''">2</MaxRetryCount>
     <HelixAccessToken Condition="'$(_UseHelixOpenQueues)' != 'true'">$(HelixApiAccessToken)</HelixAccessToken>
+    <HelixTestConfigurationFilePath>$(RepoRoot)/test-configuration.json</HelixTestConfigurationFilePath>
```

Comment:
I think by default repoRoot only includes the base path of the repo, and doesn't go into the eng folder. Unless it's being modified somewhere, Should this be

```suggestion
    <HelixTestConfigurationFilePath>$(RepoRoot)/eng/test-configuration.json</HelixTestConfigurationFilePath>
```
instead?

Reply:
Yeah I thought I typed that but must not have, thanks!